### ISSUE TITLE
The One Documentation Issue To Rule Them All

### ISSUE THREAD

**Issue by jesseduffield** [jesseduffield/lazygit#963]
Open to better ways of structuring this but I want to have a single issue for covering all the stuff we need to document better. Often an issue is at a point where it can be closed but the issue would be less likely to come up in the future if the docs were clearer.

Some things off the top of my head:
- [ ] how to edit the config
- [ ] how to customise the log graph
- [ ] how to more easily see selected lines

We should have a section (perhaps in the readme) for 'How do I...' and it can have expandable sections like so:
## How Do I...

<details>
  <summary>Edit the config</summary>
  
blah blah blah
</details>
<details>
  <summary>Customise the log graph</summary>
  
blah blah blah
</details>

**Comment by shannonxtreme** [comment]
Do you already have a technical writer doing this? I can contribute to the docs if you want; I'm a writer looking to build my portfolio with some open source work!

**Comment by jesseduffield** [comment]
Sounds good! Are you a lazygit user yourself?

**Comment by shannonxtreme** [comment]
No, I'm a filthy casual using the terminal in VS Code, but I just installed lazygit to start using it!

**Comment by jesseduffield** [comment]
Nice, let me know how you go with it, the docs could no doubt use the perspective of a newbie given most people reading the docs will be newbies as well :)

**Comment by alstocras** [comment]
bit late, but ive been using lazygit for a while now and could help with the docs
